frontmacs-editing.el

```diff
@@ -23,6 +23,9 @@
 ;; when you have a selection, typing text replaces it all.
 (delete-selection-mode t)
 
+;; remove trailing whitespace when saving
+(add-hook 'before-save-hook 'delete-trailing-whitespace)
```

Comment:
I prefer the ws-trim package with `ws-trim-level` set to 1. It will only cleanup whitespace on modified lines instead of the entire buffer